### DISCUSSION TITLE
How create Evacuation model in MESA with GIS imported map?

### DISCUSSION THREAD

**Discussion by josse7cz** [mesa/mesa#1285] [Q&A] [answered]
Hi I plese for advice. I need programming something as this. But I dont know where start.

https://nclwater.github.io/mesacat

https://user-images.githubusercontent.com/24356264/164053421-d037d2b7-cb69-4932-8eac-46700b2b1aee.mp4

/

**Comment by jackiekazil** [comment, marked as answer]
I would look at the code here: https://github.com/nclwater/mesacat 
And also checkout [mesa-geo](https://github.com/projectmesa/mesa-geo)
